app/deals/[loanCode]/_components/aml-manager.tsx

```
@@ -15,6 +15,7 @@ import {
   SelectTrigger,
   SelectValue,
 } from "@/components/ui/select";
+import { Copy, Check, Loader2, Upload } from "lucide-react";
 import { Loader2, Upload, Clock } from "lucide-react";
```

Biome (2.3.14)

[error] Shouldn't redeclare 'Loader2'. Consider to delete it or rename it.

'Loader2' is defined here:


(lint/suspicious/noRedeclare)